### PR TITLE
Test coverate info

### LINKED ISSUE #84: Test coverage info
**Is your feature request related to a problem? Please describe.**
This info can give a better idea of how much we test our implementations. Better make it more visible since we do a lot of testing anyway

**Describe the solution you'd like**
- Use integration with [code climate](https://docs.codeclimate.com/docs/github-actions-test-coverage) or any other similar tool that is compatible with GitHub Actions 
- Add a badge to the README.md

### PR DESCRIPTION
**Pull request by snk4tr**
Closes #84 

## Proposed Changes

  - Add test coverage info to the testing CI pipeline
  - Integration with [codecov](https://codecov.io/) via GitHub Actions
  - Display of coverage info with a badge in the README

### CODE REVIEW

**Comment by snk4tr** [comment]
@denproc @zakajd ready for review

File: .github/workflows/ci-testing.yml [hunk 1]
```diff
@@ -34,11 +34,20 @@ jobs:
...
+                      scikit-image
       - name: Test with pytest
         run: |
-          pip install pytest
-          pip install tensorflow
-          pip install libsvm
-          pip install pybrisque
-          pip install scikit-image
-          pytest
+          pytest -x --cov=./ --cov-report=xml
```
Comment (denproc) [inline .github/workflows/ci-testing.yml:45]: I suggest to show the coverage only for `piq` code without tests.
```suggestion
          pytest -x --cov=./piq --cov-report=xml
```

File: codecov.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      macro: no
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: false  # if true: only post the comment if coverage changes
+  require_base: no        # [yes :: must have a base report to post]
+  require_head: no        # [yes :: must have a head report to post]
+  branches:               # branch names that can post comment
+    - "master"
```
Comment (denproc) [inline codecov.yml:38]: All branches except `master` should be available here In order to enable comments in PR according to [source](https://docs.codecov.io/docs/pull-request-comments#branches). The first report should appear on this PR.

File: codecov.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      default:
+        # basic
+        target: auto
+        threshold: 5%
+        base: auto
+        flags:
+          - unit
+        # advanced
+        branches:
+          - master
```
Comment (denproc) [inline codecov.yml:16]: All branches should be able to trigger the Codecov action. 
As other CI testing is triggered only for master commit and pull request actions, Codecov action will be triggered in the same way.

File: codecov.yml (hunk 2 above)
Comment (snk4tr) [reply codecov.yml:38]: I was curious why there are no comments in this PR. Let's try this 👍

File: .github/workflows/ci-testing.yml (hunk 1 above)
Comment (snk4tr) [reply .github/workflows/ci-testing.yml:45]: Let's try this 😄

**Comment by denproc** [comment]
@snk4tr We have to install [Codecov app](https://github.com/apps/codecov) into our repo or add [Codecov bot](https://docs.codecov.io/docs/team-bot).
The bot can be added only by the owner of the organisation. I assume the same on case of app installation.

**Pull request merged**